evaluation/swe_bench/run_infer.py

```diff
@@ -488,6 +496,7 @@ def filter_dataset(dataset: pd.DataFrame, filter_column: str) -> pd.DataFrame:
     llm_config = None
     if args.llm_config:
         llm_config = get_llm_config_arg(args.llm_config)
+        llm_config.log_completions = True
```

Comment:
Hah. Sneaky, but probably good to have. 😅

Reply:
i've once been told during one of my internship that "disk are cheap" 🤣